### PR TITLE
Cookie fixes

### PR DESCRIPTION
**Pull request by Dankgen**
# Changes to cookie usage
- The culprit to the cookie header not passing automatically was CORS settings
- There is now a new env variable in the API for this
- You no longer need to manually pass the cookie, it will be grabbed in the api with request.cookies.userCookie using the added cookie-parser middleware

# Minor changes regarding likes/comments
- Comments now show up instantly after posting them
- Likes instantly reflect on like count
- Comment/Like count now accurately reflect on their post

### CODE REVIEW

File: harmony-api/.env.example [hunk 1]
```diff
@@ -1,6 +1,9 @@
 # Port that this server will be accessible on
 PORT=8081
 
+# Endpoint for the frontend
+FRONT_END_BASE_ENDPOINT=http://localhost:8080
+
```
Comment (Developik) [inline harmony-api/.env.example:6]: I guess that I will need to add variable for this in cd

Comment (Developik) [reply harmony-api/.env.example:6]: @Dankgen I will add a variable tonight and then merge yours afterwards

Comment (Dankgen) [reply harmony-api/.env.example:6]: Yeah, Its needed for the CORS setting

Comment (Dankgen) [reply harmony-api/.env.example:6]: > @Dankgen I will add a variable tonight and then merge yours afterwards

Given that you guys find the PR satisfactory, you can merge it after you set it up

File: harmony-api/src/repos/postRepo.ts
Comment (Dankgen) [inline harmony-api/src/repos/postRepo.ts:1]: If anybody knows a better way of getting the number of comments/likes in a query like this, please do tell me 😅

File: harmony-api/src/repos/userRepo.ts
Comment (Dankgen) [inline harmony-api/src/repos/userRepo.ts:1]: Was returning the hashed passwords, Dont want that

**Review by Developik — APPROVED** [review]
Everything looks good to me

**Pull request merged**